_data/saswat01/saswat01.md/Readme

```diff
@@ -0,0 +1,7 @@
+name: SASWAT NANDA
+institution: Veer Surendra Sai University of Technology
+profile_pic: saswat_profile.jpg
+quote: My life is mine to remember
+github_user: saswat01
+
+
```

Comment:
Add --- to the beginning and end of your .md. Don't change the template.

Reply:
Thank you for identifying problem. I have made the changes please verify.